### PR TITLE
DT-778 ui

### PR DESCRIPTION
**Pull request by mikehalmamoj**
Ripped off the static resources from auth server and add a simple UI to list all messages currently in memory.

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/hmpps/offenderevents/resource/MessagesController.kt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+@Controller
+class MessagesController {
+
+  @Autowired
+  private lateinit var offenderEventStore: OffenderEventStore
+
+  @GetMapping("/messages")
+  fun messages(): ModelAndView {
+    return ModelAndView("index", mutableMapOf("storedMessages" to offenderEventStore.getAllMessages()))
```
Comment (mikehalmamoj) [inline src/main/kotlin/uk/gov/justice/hmpps/offenderevents/resource/MessagesController.kt:17]: First attempt is just to get a very basic UI working - will iterate over this

File: src/test/kotlin/uk/gov/justice/hmpps/offenderevents/service/UiIntegrationTest.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  @Test
+  fun `Should Display captured event`() {
+    val message = "/messages/externalMovement.json".readResourceAsText()
+    awsSqsClient.sendMessage(queueUrl, message)
+    `Wait for empty queue`()
+
+    val response = URL("$baseUrl/messages").readText()
+
+    assertThat(response).contains("EXTERNAL_MOVEMENT_RECORD-INSERTED")
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/hmpps/offenderevents/service/UiIntegrationTest.kt:29]: I considered diving into Fluentlenium for these tests, but as this is a dev tool I'm not sure there is any value as I won't be producing a huge test suite.  Therefore the plan is to keep the testing simple and light-touch - just want to prove that the app stands up.

If anyone has better ideas for super simple UI testing then I'm all ears.

File: src/main/resources/static/css/app-ie8.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
+#header-container {
```
Comment (mikehalmamoj) [inline src/main/resources/static/css/app-ie8.css:1]: All of the static content was pinched out of the Auth server without applying much thought...

File: src/main/resources/templates/fragments/layout.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  <title layout:title-pattern="$LAYOUT_TITLE - $CONTENT_TITLE"
+         th:text="${(param.error != null || error != null) ? 'Error: ' : ''} + 'HMPPS Digital Services'">
+    HMPPS Digital Services
+  </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <meta name="theme-color" content="#0b0c0c"/>
+
+  <!-- Google Tag Manager -->
+  <script th:if="${@environment.getProperty('application.google.tag.id')} != null">(function(w, d, s, l, i) {
```
Comment (andymarke) [inline src/main/resources/templates/fragments/layout.html:17]: I wonder if anything bad will happen if we have this GA stuff

File: src/test/kotlin/uk/gov/justice/hmpps/offenderevents/service/UiIntegrationTest.kt (hunk 2 above)
Comment (andymarke) [reply src/test/kotlin/uk/gov/justice/hmpps/offenderevents/service/UiIntegrationTest.kt:29]: This looks good for this DEV tool

File: src/main/resources/templates/fragments/layout.html (hunk 4 above)
Comment (mikehalmamoj) [reply src/main/resources/templates/fragments/layout.html:17]: Good catch - I'll remove in the next PR.

**Pull request merged**